ctrd/utils.go

```diff
@@ -25,6 +25,7 @@ import (
 )
 
 // NewDefaultSpec new a template spec with default.
+// TODO(huamin): remove NewDefaultSpec, pouch use default spec defined in oci package
 func NewDefaultSpec(ctx context.Context, id string) (*specs.Spec, error) {
```

Comment:
since you created `NewDefaultSpec` in pouch cri package, could we remove this functionality?

Reply:
Remove it when we test enough.

Reply:
Could we update the `oci.NewDefaultSpec` when we meet issue? Because I cannot see any timepoint about `test enough'

Reply:
Already removed, @fuweid .